api/v1/mapping/src/commonMain/kotlin/ApiMappings.kt

```diff
@@ -619,7 +619,7 @@ fun RuleViolation.mapToApi() = ApiRuleViolation(
     rule = rule,
     id = id?.mapToApi(),
     license = license,
-    licenseSource = licenseSource,
+    licenseSource = licenseSources.firstOrNull(),
```

Comment:
I believe this should be marked with a `TODO` in the code *and* we should create a follow-up issue from these lines once merged to not forget about adding support for multiple license sources going forward.